examples/basic/annotations/band.py

```diff
@@ -1,9 +1,11 @@
-''' A demonstration of the Band annotation
+''' An interactive numerical band plot based on simple Python array of data.
+    It is a combination of scatter plots and line plots added with a band of covered area.
+    The line passes through the mean of the area covered by the Band.
 
 .. bokeh-example-metadata::
-    :apis: bokeh.models.Band, bokeh.models.ColumnDataSource
+    :apis: bokeh.models.Band, bokeh.models.ColumnDataSource, bokeh.plotting.figure, bokeh.io.show,
     :refs: :ref:`ug_basic_annotations_bands`
-    :keywords: band
+    :keywords: bandplot, figure, scatter, line, Band, add_layout, rolling
```

Comment:
I'd suggest paring this down a bit:
```suggestion
    :keywords: figure, scatter, line, band, layout
```
Unfortunately we don't yet have any actual standards for what goes in "keywords" but so far they have usually been more general than specific APIs. @tcmetzger maybe we can sit down sometime soon and actually try to define style guides for keyords?